clippy_utils/src/msrvs.rs

```diff
@@ -25,7 +25,7 @@ msrv_aliases! {
     1,40,0 { MEM_TAKE, NON_EXHAUSTIVE, OPTION_AS_DEREF }
     1,38,0 { POINTER_CAST }
     1,37,0 { TYPE_ALIAS_ENUM_VARIANTS }
-    1,36,0 { ITERATOR_COPIED }
+    1,36,0 { ITERATOR_COPIED, PTR_CAST }
```

Comment:
What method is this? I thought it was [this](https://doc.rust-lang.org/std/primitive.pointer.html#method.cast), but it looks like this was added at 1.38.0.

Reply:
That is the right method. I don't know where I got 1.36 from.

Reply:
Okay. Can `1,38,0 { POINTER_CAST }` be used for msrv?